packages/astro/e2e/fixtures/actions-blog/src/actions/index.ts

```diff
@@ -15,16 +15,16 @@ export const server = {
 			handler: async ({ postId }) => {
 				await new Promise((r) => setTimeout(r, 500));
 
-				const { likes } = await db
+				const result  = await db
 					.update(Likes)
 					.set({
-						likes: sql`likes + 1`,
+						likes: sql`${Likes.likes} + 1`,
```

Comment:
Hm, this doesn't seem right. Any reason we stopped incrementing from sql for this test?